src/test/java/com/endava/endabank/utils/TestUtils.java

```diff
@@ -0,0 +1,197 @@
+    public static final String SECRET_DUMMY = "ZHVtbXkgdmFsdWUK";
+
+    public static User getUserAdmin() {
+        Role adminRole = TestUtils.adminRole();
+        IdentifierType cc = TestUtils.identifierTypeCC();
+        return User.builder().
+                id(1).
+                email("admin@endava.com").
+                phoneNumber("3212312321").
+                identifier("1001000000").
+                firstName("Endava").
+                lastName("Admin").
+                password("$2a$10$bUcuzJbChZheTqPERIqk3u7COWhAF1CV6OU.LUkCG6iZYRhXydRqW").
+                isEmailVerified(true).
+                isApproved(true).
+                role(adminRole).
+                identifierType(cc).
+                bankAccounts(new ArrayList<>()).build();
+    }
+
+    public static UserToApproveAccountDto getUserApprovedAccountDto() {
+        return UserToApproveAccountDto.builder().
+                id(1).
+                email("user@endava.com").
+                isApproved(true).
+                firstName("Endava").
+                lastName("User").build();
+    }
+
+    public static UserToApproveAccountDto getUserNotAprrovedAccountDto() {
+        UserToApproveAccountDto userToApproveAccountDto = getUserApprovedAccountDto();
+        userToApproveAccountDto.setApproved(false);
+        return userToApproveAccountDto;
+    }
+
+    public static User getUserNotAdmin() {
+        Role userRole = TestUtils.userRole();
+        IdentifierType cc = TestUtils.identifierTypeCC();
+        return User.builder().
+                id(1).
+                email("user@endava.com").
+                phoneNumber("3210000000").
+                identifier("1001000000").
+                firstName("Endava").
+                lastName("User").
+                password("$2a$10$bUcuzJbChZheTqPERIqk3u7COWhAF1CV6OU.LUkCG6iZYRhXydRqW").
+                isApproved(true).
+                role(userRole).
+                identifierType(cc).
+                bankAccounts(new ArrayList<>()).build();
+    }
+
+    public static User getUserNotAdminNonApproved() {
+        User user = getUserNotAdmin();
+        user.setIsApproved(false);
+        return user;
+    }
+
+
+    public static Role adminRole() {
+        return new Role(1, "ROLE_ADMIN", new ArrayList<>(), adminPermission());
+    }
+
+    public static HashSet<Permission> adminPermission() {
+        Permission permission = new Permission(1, "Dummy-permission", new ArrayList<>());
+        HashSet<Permission> set = new HashSet<>();
+        set.add(permission);
+        return set;
+    }
+
+    public static Role userRole() {
+        return new Role(2, "ROLE_USER", new ArrayList<>(), new HashSet<>());
+    }
+
+    public static IdentifierType identifierTypeCC() {
+        return new IdentifierType(1, "CC", new ArrayList<>());
+    }
+
+    public static UserRegisterDto getUserRegisterDto() {
+        return UserRegisterDto.builder().
+                email("user@endava.com").
```

Comment:
Hi,

This repository is public, so please do not use Endava information or sensitive information.